### ISSUE TITLE
Inline frame missing a text alternative (IU Accessibility Mandate)

### ISSUE THREAD

**Issue by mdalmau** [IUBLibTech/newton_chymistry#136]
6 occurrences in one page (https://newton.dlib.indiana.edu/page/chem-lab). The embedded iFrame needs a title attribute. For example: `<iFrame src="url" title="Embedded video player demonstrating the transmutation of silver into gold"> <iFrame>`

Related to [https://github.com/IUBLibTech/newton_chymistry/issues/137](https://github.com/IUBLibTech/newton_chymistry/issues/137) (same HTML page with issues)

--------------------------
IU has a mandate that all web sites published on IU servers must meet WCAG I and II Accessibility requirements. If the requirements are NOT met, sites will be taken down. To learn more about this initiative, visit the [Digital Strategy Initiative](https://ucm.iu.edu/digital-strategy/index.html) web site.

- [x] Ask Bill who should be added to the SiteImprove tool

**Comment by mdalmau** [comment]
@tubesoft : At the Newton meeting today, Bill said you would be working on these accessibility issues. I requested SiteImprove access for you. Once you have access, you will login using your @iu credentials. If you want to read more about SIteImprove and accessibility, LOGIN to the KB and [start here](https://kb.iu.edu/d/webs). 

Could you have a look at this issue and confirm it's something you can fix? Once you confirm, we can figure out a testing workflow.

**Comment by tubesoft** [comment]
@mdalmau, I fixed the issues and committed, as you can see above.

**Comment by mdalmau** [comment]
Since this issue is now part of https://github.com/IUBLibTech/newton_chymistry/pull/146, I will mark this issue as "done" so I can better see the remaining accessibility issues!
